express-hello-world/index.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+
+app.use(function (req, res) {
+  // eslint-disable-next-line no-console
+  console.log('Method property of req:', req.method);
+  const sentMessage = res.send('Hello World');
+  // eslint-disable-next-line no-console
+  console.log('Send method of res:', sentMessage);
```

Comment:
Not a huge deal in this case but we usually don't want any code after calling `res.send` that should be the last thing in the callback.